aten/src/ATen/Dispatch.h

```diff
@@ -220,6 +220,23 @@ inline void deprecated_AT_DISPATCH_ALL_TYPES_AND_HALF_AND_COMPLEX() {}
     }                                                                        \
   }()
 
+#define AT_DISPATCH_INTEGRAL_TYPES_AND_BOOL(TYPE, NAME, ...)                 \
```

Comment:
we've shied away from these crazily-named dispatch macros -- the ones that continue to be in here are here for BC reasons.

Can we call make a `AT_DISPATCH_INTEGRAL_TYPES_AND` macro and make it follow `AT_DISPATCH_ALL_TYPES_AND` (i.e. don't call `::detail::scalar_type`)?

Reply:
I just realized that this is actually not used in this PR. I have removed this macro.